src/cow-react/modules/wallet/web3-react/connectors/Injected/index.tsx

```diff
@@ -126,7 +126,7 @@ export class InjectedWallet extends Connector {
 
     return Promise.all([
       this.provider.request({ method: 'eth_chainId' }) as Promise<string>,
-      this.provider.request({ method: 'eth_accounts' }) as Promise<string[]>,
+      this.provider.request({ method: 'eth_requestAccounts' }) as Promise<string[]>,
```

Comment:
I guess we should use both because some wallets support eth_accounts, and some eth_requestAccounts.

Add it as a fallback:
```
this.provider.request({ method: 'eth_accounts' })
.catch(() => this.provider.request({ method: 'eth_requestAccounts' }))
```

Reply:
Good point, updated 👍

Reply:
Shouldn't it be first `eth_requestAccounts` which is the official new way? https://eips.ethereum.org/EIPS/eip-1102

They also mention in https://eips.ethereum.org/EIPS/eip-1193

![image](https://user-images.githubusercontent.com/2352112/228599574-fbdf369c-65ef-499b-bc12-4f22077c8228.png)

Reply:
@anxolin I've put `eth_requestAccounts` first